pkg/webreg/webreg.go

```diff
@@ -328,6 +333,195 @@ const templateDefinitions = `
 {{ end }}
 `
 
+const helpPage = `
+<h1><a href="/">Openshift CI Step Registry</a></h1>
+<h1>What is the Multi-Stage Test and the Test Step Registry?</h1>
+
+<p>
+The multistage test style in the ci-operator is a modular test design that
+allows users to create new tests by combining smaller, individual test steps.
+These individual tests can be put into a shared registry that other tests can
```

Comment:
It feels like the sentence should start "These individual steps" instead. If the intention is really to speak about "tests", then I suggest to remove the _individual_ that makes it connected to the "individual test step" from the previous sentence.